### PR TITLE
fix set LODs will change the input's variable & refine function name

### PR DESCRIPTION
**Pull request by bofeng-song**
Re: #

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Comment by minggo** [comment]
I think should modify setter instead of getter.

**Comment by bofeng-song** [comment]
> I think should modify setter instead of getter.

modified

File: editor/inspector/components/lodGroup/index.js [hunk 1]
```diff
@@ -253,9 +253,21 @@ exports.ready = function() {
                 const that = this;
                 const LODs = that.dump.value.LODs.value;
                 if (range === 'min') {
-                    return LODs[index + 1] ? LODs[index + 1].value.screenUsagePercentage.value * 100 : 0;
+                    const min = LODs[index + 1] ? LODs[index + 1].value.screenUsagePercentage.value * 100 : 0;
+                    // 如果 value < min，设置值为 min，避免影响到其他 lod
```
Comment (minggo) [inline editor/inspector/components/lodGroup/index.js]: Please don't use Chinese.

File: editor/inspector/components/lodGroup/index.js [hunk 2]
```diff
@@ -253,9 +253,21 @@ exports.ready = function() {
...
+                    if (LODs[index].value.screenUsagePercentage.value < min / 100) {
```
Comment (minggo) [inline editor/inspector/components/lodGroup/index.js]: Can cache the result of `min / 100`.

File: editor/inspector/components/lodGroup/index.js [hunk 3]
```diff
@@ -253,9 +253,21 @@ exports.ready = function() {
...
+                        LODs[index].value.screenUsagePercentage.value = min / 100;
+                        that.updateDump(LODs[index].value.screenUsagePercentage);
+                    }
+                    return min;
                 } else if (range === 'max') {
-                    return LODs[index - 1] ? LODs[index - 1].value.screenUsagePercentage.value * 100 : 100;
+                    const max = LODs[index - 1] ? LODs[index - 1].value.screenUsagePercentage.value * 100 : 100;
+                    // 如果 value > max，设置值为 max，避免影响到其他 lod
+                    if (LODs[index].value.screenUsagePercentage.value > max / 100) {
```
Comment (minggo) [inline editor/inspector/components/lodGroup/index.js]: Can cache the result of `max / 100`.

Comment (minggo) [reply editor/inspector/components/lodGroup/index.js]: And also can cache `LODs[index].value.screenUsagePercentage`.

**Review by minggo — APPROVED** [review]

**Pull request merged**